components/IscnRegisterForm.vue

```diff
@@ -1560,6 +1561,7 @@ estimation,
       console.error(err)
     } finally {
       this.isOpenQuitAlertDialog = false
+      this.fetchWalletBalance()
```

Comment:
Maybe replace `getAccountBalance` call with `fetchWalletBalance` and `getBalance` from store?